Mathlib/MeasureTheory/Measure/NullMeasurable.lean

```diff
@@ -280,6 +280,12 @@ theorem measure_union₀' (hs : NullMeasurableSet s μ) (hd : AEDisjoint μ s t)
 theorem measure_add_measure_compl₀ {s : Set α} (hs : NullMeasurableSet s μ) :
     μ s + μ sᶜ = μ univ := by rw [← measure_union₀' hs aedisjoint_compl_right, union_compl_self]
 
+lemma measure_of_measure_compl_eq_zero₀ (hs : NullMeasurableSet s μ) (h : μ sᶜ = 0) :
```

Comment:
You shouldn't need `hs` here: `s^c` is contained in a measurable set of measure zero, so `s` contains a measurable set of full measure.

Reply:
And in fact you already know that `s` is null-measurable, as `s^c` is by `h`...

Reply:
Whoops yeah 🙈 

I was trying to find something along those lines, but I think I got lost when trying a separate generalisation (which didn't work out) simultaneously